src/modules/validation/ISingleSignerValidation.sol

```diff
@@ -7,11 +7,11 @@ interface ISingleSignerValidation is IValidation {
     /// @notice This event is emitted when Signer of the account's validation changes.
     /// @param account The account whose validation Signer changed.
     /// @param entityId The entityId for the account and the signer.
-    /// @param previousSigner The address of the previous signer.
     /// @param newSigner The address of the new signer.
+    /// @param previousSigner The address of the previous signer.
     event SignerTransferred(
-        address indexed account, uint32 indexed entityId, address previousSigner, address newSigner
-    );
+        address indexed account, uint32 indexed entityId, address indexed newSigner, address indexed previousSigner
+    ) anonymous;
```

Comment:
Do we need to index signers?
They would cost probably at least 700 more gas for those two.
In the real world use case, how many times would a signer be transfered?

Reply:
I think it'd be less than 700 since we make the event anonymous. But since we already index the validation ID (acccount, entityid) why do we need to index the signers to find previously set owners @adamegyed ?

Reply:
So the improvement in devX is from get all account SignerTransfer events and filter out new signer address => get the event where the new signer is 0x0000...
That is a lot more convenient.
Sounds good on keeping `newSigner` and rid `previousSigner` index.

Reply:
Ok, I updated the event to have the following signature:
```solidity
event SignerTransferred(address indexed account, uint32 indexed entityId, address indexed newSigner, address previousSigner) anonymous;
```